faust-src/Makefile

```diff
@@ -2,8 +2,9 @@ h headers :
 	install -d headersdir
 	$(MAKE) DEST='headersdir/' ARCH='faust2header.cpp' -f Makefile.headers
 
+# install sets the "executable" bit, so we use cp instead:
 ih hi installheaders : headers
-	install headersdir/*.h ../src/
+	/bin/cp -p headersdir/*.h ../src/
```

Comment:
FYI install has a "-m" parameter to set the specific mode

Reply:
Thanks -  I switched to that and realized the headers should be installed as readonly since they are derived source.